<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="dump_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="dump_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="dump_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="dump_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="bank">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stack_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="logo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="bank_options">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="table">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-9</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <zorder>table</zorder>
   <zorder>dump_4</zorder>
   <zorder>dump_3</zorder>
   <zorder>dump_2</zorder>
   <zorder>dump_1</zorder>
   <zorder>bank</zorder>
   <zorder>stack_7</zorder>
   <zorder>stack_6</zorder>
   <zorder>stack_5</zorder>
   <zorder>stack_4</zorder>
   <zorder>stack_3</zorder>
   <zorder>stack_2</zorder>
   <zorder>stack_1</zorder>
   <zorder>logo</zorder>
   <zorder>bank_options</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionDeal"/>
    <addaction name="separator"/>
    <addaction name="action1_Card"/>
    <addaction name="action3_Card"/>
    <addaction name="separator"/>
    <addaction name="action3_Rounds"/>
    <addaction name="action5_Rounds"/>
    <addaction name="actionUnlimited_rounds"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDeal">
   <property name="text">
    <string>Deal</string>
   </property>
  </action>
  <action name="action1_Card">
   <property name="text">
    <string>1 Card</string>
   </property>
  </action>
  <action name="action3_Card">
   <property name="text">
    <string>3 Card</string>
   </property>
  </action>
  <action name="action3_Rounds">
   <property name="text">
    <string>3 Rounds</string>
   </property>
  </action>
  <action name="action5_Rounds">
   <property name="text">
    <string>5 Rounds</string>
   </property>
  </action>
  <action name="actionUnlimited_rounds">
   <property name="text">
    <string>Unlimited rounds</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
